designer/server/src/test/scala/pl/touk/nussknacker/ui/api/ProcessActivitySpec.scala

```diff
@@ -0,0 +1,361 @@
+package pl.touk.nussknacker.ui.api
+
+import io.restassured.RestAssured.`given`
+import io.restassured.http.ContentType
+import io.restassured.module.scala.RestAssuredSupport.AddThenToResponse
+import io.restassured.response.ValidatableResponse
+import org.scalatest.freespec.AnyFreeSpecLike
+import pl.touk.nussknacker.engine.api.process.ProcessName
+import pl.touk.nussknacker.engine.build.ScenarioBuilder
+import pl.touk.nussknacker.engine.canonicalgraph.CanonicalProcess
+import pl.touk.nussknacker.test.{NuRestAssureExtensions, NuRestAssureMatchers, RestAssuredVerboseLogging}
+import pl.touk.nussknacker.ui.api.helpers.TestCategories.Category1
+import pl.touk.nussknacker.ui.api.helpers.{
+  NuItTest,
+  NuScenarioConfigurationHelper,
+  TestCategories,
+  TestProcessingTypes,
+  WithMockableDeploymentManager
+}
+
+import java.nio.charset.StandardCharsets
+import java.util.UUID
+
+class ProcessActivitySpec
+    extends AnyFreeSpecLike
+    with NuItTest
+    with WithMockableDeploymentManager
+    with NuScenarioConfigurationHelper
+    with NuRestAssureExtensions
+    with NuRestAssureMatchers
+    with RestAssuredVerboseLogging {
+
+  val processName    = ProcessName(UUID.randomUUID().toString)
+  val commentContent = "test message"
+
+  val process: CanonicalProcess = ScenarioBuilder
+    .streaming(processName.value)
+    .source("sourceId", "barSource")
+    .emptySink("sinkId", "barSink")
+
+  "The process activity endpoint should" - {
+    "get process activity" - {
+      "authenticated should" - {
+        "return empty comments and attachment for existing process without them" in {
+          createSavedProcess(process, category = Category1, TestProcessingTypes.Streaming)
```

Comment:
We have the `applicationConfiguration` ext to write the rest assure code like this:

```scala
          given
            .applicationConfiguration {
              createSavedProcess(process, TestCategories.Category1, TestProcessingTypes.Streaming)
            }
            .basicAuth("allpermuser", "allpermuser")
```

The creation of the process is in the `given` section. 
Maybe we should rename `applicationConfiguration` to sth more meaningful. It's not an application configuration but preparing a proper app state before testing.

Reply:
Done